robot/markers.py

```diff
@@ -117,12 +149,18 @@ def is_token_marker(self) -> bool:
     @property
     def polar(self) -> PolarCoord:
         """
-        The position of the marker in the polar co-ordinates.
+        Deprecated: the position of the marker in legacy "polar" co-ordinates.
```

Comment:
If `@deprecated` was part of the standard library I'd recommend using it here. However it seems it would require pulling in a [third party library](https://pypi.python.org/pypi/deprecation) and we're raising a warning manually anyway, so its up to you whether you want to do this or not - I'm happy either way.